### PR TITLE
ENG-909

### PR DESCRIPTION
**Pull request by sblack4**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -1,11 +1,84 @@
...
+  type        = string
+}
+
+variable "identity_source" {
+  default     = "method.request.header.x-api-key"
+  description = <<EOT
+The source of the identity in an incoming request.
+For REQUEST type, this may be a comma-separated list of values, including headers, query string parameters and stage variables - e.g.
+`"method.request.header.SomeHeaderName,method.request.querystring.SomeQueryStringName,stageVariables.SomeStageVariableName"`
+EOT
```
Comment (smiller171) [inline variables.tf]: Push this below `type` since it's a multi-line value

File: variables.tf [hunk 2]
```diff
@@ -1,11 +1,84 @@
...
   description = "Moniker to apply to all resources in the module"
   type        = string
 }
 
+variable "provider_arns" {
+  default     = []
+  description = <<EOT
+A list of the Amazon Cognito user pool ARNs. Each element is of this format:
+`arn:aws:cognito-idp:{region}:{account_id}:userpool/{user_pool_id}`.
+EOT
```
Comment (smiller171) [inline variables.tf:52]: push this below `type` since it's multi-line

File: variables.tf [hunk 3]
```diff
@@ -1,11 +1,84 @@
...
   type        = map(string)
 }
+
+variable "types" {
+  default     = ["EDGE"]
+  description = <<EOT
+A list of endpoint types. This resource currently only supports managing a single value.
+Valid values: EDGE, REGIONAL or PRIVATE. If unspecified, defaults to EDGE. Must be declared as REGIONAL in non-Commercial partitions.
+Refer to the documentation for more information on the difference between edge-optimized and regional APIs.
+EOT
```
Comment (smiller171) [inline variables.tf:79]: Push this below `type` since it's multi-line

**Review by smiller171 — CHANGES_REQUESTED** [review]
Very minor changes

File: variables.tf (hunk 1 above)
Comment (sblack4) [inline variables.tf:26]: true dat

**Review by smiller171 — APPROVED** [review]
lgtm

**Pull request merged**